_articles/legal.md

```diff
@@ -84,7 +86,7 @@ For example, as your project grows it adds dependencies or users, or your compan
 
 **Your project's existing copyright holders.** If you're the sole contributor to your project then either you or your company is the project's sole copyright holder. You can add or change to whatever license you or your company wants to. Otherwise there may be other copyright holders that you need agreement from in order to change licenses. Who are they? People who have commits in your project is a good place to start. But in some cases copyright will be held by those people's employers. In some cases people will have only made minimal contributions, but there's no hard and fast rule that contributions under some number of lines of code are not subject to copyright. What to do? It depends. For a relatively small and young project, it may be feasible to get all existing contributors to agree to a license change in an issue or pull request. For large and long-lived projects, you may have to seek out many contributors and even their heirs. Mozilla took years (2001-2006) to relicense Firefox, Thunderbird, and related software.
```

Comment:
I have a project which helps with identifying the existing copyright holders in your code, but I don't want to include it without prior discussion to avoid "conflicts of interest"/"self-promotion" concerns.

https://github.com/thehale/git-authorship

Reply:
I don't think this would present a conflict of interest, if you still want to include it please feel free to do so!